### PR TITLE
Upgrade deps to enable postcss 8

### LINKED ISSUE #324: PostCSS 8
PostCSS 8 was released. It has no breaking changes related to `rollup-plugin-postcss`, most of changes are for plugins. [Full changelog](https://github.com/postcss/postcss/releases/tag/8.0.0).

- [ ] Move `postcss` to `peerDependencies`
- [ ] Update `postcss` version

### PR DESCRIPTION
**Pull request by isaacl**
Fixes #324

### CODE REVIEW

**Comment by ai** [comment]
LGTM

**Comment by softbeehive** [comment]
Hey folks, is there a blocker? I wonder why it hasn't been approved yet

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rollup-plugin-postcss",
-  "version": "3.0.0",
+  "version": "3.1.8",
```
Comment (softbeehive) [inline package.json:3]: Do you need to bump version manually? 3.1.8 was published in August, I assume next release including these changes will actually have a different version

**Comment by spacedawwwg** [comment]
Good to see this one go in

**Comment by danielbarion** [comment]
I'm very excited and waiting for this PR, nice!

**Comment by SASUKE40** [comment]
Is this PR a duplicate of the previous one ？
https://github.com/egoist/rollup-plugin-postcss/pull/325

**Comment by ai** [comment]
It is an improvement for #325

We need to update `postcss-load-config` to be able to load PostCSS 8 plugins.

**Comment by SASUKE40** [comment]
I upgraded it in another pr, and migrated the noop postcss plug-in.

**Comment by wardpeet** [comment]
Thanks for your PR and sorry for the delay, it seems that #342 already fixes this.

**Pull request closed without merging**